packages/web-app/src/modules/notifications/models/NotificationMessage.tsx

```diff
@@ -52,3 +54,58 @@ export interface NotificationMessage {
    */
   onClick?: () => void
 }
+
+/** A resource that represents an action that acknowledges a notification. */
+export interface AcknowledgeNotificationAction {
+  /** The title. */
+  title: string;
+}
+/** A resource that represents an action that dismisses a notification. */
+export interface DismissNotificationAction {
+  /** The title. */
+  title: string;
+}
+/** A resource that represents an action that opens a link in the default browser. */
+export interface OpenLinkNotificationAction {
+  /** The title. */
+  title: string;
+  /** The link. */
+  link: string;
+}
+
+/** A resource that represents an in-app notification action. */
+export interface NovuNotificationAction {
```

Comment:
Seems the naming is misleading here
the interface below has nothing common with novu notification template 
```suggestion
export interface NotificationAction {
```